TravelAgency-Persistence/src/main/java/cz/muni/fi/pa165/travelAgency/persistence/dao/ExcursionDao.java

```diff
@@ -45,24 +46,15 @@
      *
      * @return all excursions
      */
-    public Set<Excursion> findAllExcursions();
+    public List<Excursion> findAllExcursions();
     
     /**
      * List all excursions in given Trip
      *
      * @param trip trip for which we want to find excursions
      * @return all Excursions in Trip
      * @throws IllegalArgumentException in case trip is not found
-     */
-    //public Set<Excursion> findAllExcursionsInTrip(Trip trip);
+     */    
     
-    /**
-     * Find all excursions in a reservation
-     *
-     * @param reservation Reservation for which we want to find the excursions
-     * @return all excursions in reservation
-     */
-    public Set<Excursion> findAllExcursionsInReservation(Reservation reservation);
-
     
```

Comment:
vyzera to, ze ti chybaju posledne dva riadky, alebo tie metody si chcela zamerne odstranit?